DXMainClient/DXGUI/Generic/MainMenu.cs

```diff
@@ -385,9 +407,9 @@ private void SettingsSaved(object sender, EventArgs e)
                 ProgramConstants.PLAYERNAME = UserINISettings.Instance.PlayerName;
 
             if (UserINISettings.Instance.DiscordIntegration)
-                discordHandler?.Connect();
+                discordHandler.Connect();
             else
-                discordHandler?.Disconnect();
```

Comment:
If `ClientConfiguration.Instance.DiscordAppId` is empty, the `discordHandler` would be null. So why replace `?.` with `.`?

Reply:
See comment above.